a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -177,6 +179,11 @@ def _set(self, set_method, c, context, listener):
         if hasattr(listener, 'aflex'):
             template_args["aflex-scripts"] = listener.aflex
 
+        if "protocol" in vport_defaults:
```

Comment:
Shouldn't this be done higher up as the protocol, which will soon be set, effects above logic?

Reply:
This ensures the override from vport expressions applies so we still apply correct HTTPS logic.